shell/bashrc

```diff
@@ -319,7 +324,6 @@ function phpbrew ()
             unset PHPBREW_PATH
             eval `$BIN env`
             __phpbrew_set_path
-            echo "phpbrew is turned off."
```

Comment:
I agree. However, the change in the command itself is a much greater BC break which is probably worth documenting somewhere (cannot find any existing documentation). I.e. if somebody has a scripted command like `phpbrew each ext install xdebug` (assuming it works currently like this), they will have to update it to `phpbrew each phpbrew ext install xdebug`.

Do you have any suggestions on documenting the change?